bundles/EcommerceFrameworkBundle/FilterService/FilterType/MultiSelectCategory.php

```diff
@@ -25,9 +26,11 @@ public function getFilterValues(AbstractFilterDefinitionType $filterDefinition,
         $values = [];
 
         $availableRelations = [];
-        if ($filterDefinition->getAvailableCategories()) {
-            foreach ($filterDefinition->getAvailableCategories() as $rel) {
-                $availableRelations[$rel->getId()] = true;
+        if (method_exists($filterDefinition, 'getAvailableCategories')) {
```

Comment:
we should check `$filterDefinition instanceof \Pimcore\Model\DataObject\Fieldcollection\Data\FilterCategoryMultiSelect`, other wise throw `InvalidConfigException`